src/ProjectService/Controllers/ImageController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using LT.DigitalOffice.Kernel.Enums;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.ProjectService.Business.Commands.Image.Interfaces;
+using LT.DigitalOffice.ProjectService.Business.Commands.Task.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Net;
+
+namespace LT.DigitalOffice.ProjectService.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class ImageController : ControllerBase
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
do not set status code in controller, set in command

Reply:
done